Hadrons/Modules/MContraction/Meson.hpp

```diff
@@ -249,7 +262,17 @@ void TMeson<FImpl1, FImpl2>::execute(void)
             }
         }
     }
-    saveResult(par().output, "meson", result);
+
+    
+    if (par().output.length())
```

Comment:
I would prefer the boolean `empty()` to be used

Reply:
Now updated to use `empty()` in the two places where `length()` was previously used